src/reducers/app/thunks.ts

```diff
@@ -51,6 +51,7 @@ const {
 } = ConseilQueryBuilder;
 
 const CACHE_TIME = 432000000; // 5*24*3600*1000
+const CACHE_VERSION = 4;
```

Comment:
cache version can be also passed as a global variable https://github.com/Cryptonomic/Arronax/blob/develop/package.json#L62
and used like here 
https://github.com/Cryptonomic/Arronax/blob/master/src/components/Footer/index.tsx#L38

Reply:
I don't love the idea of using env vars for things, putting it in a config file would be easier to manager.